INDEX

```diff
@@ -216,6 +216,14 @@
         "official": false,
         "primary_pkg": "openvpn"
     },
+  "OpenSpeedTest-Server": {
+    "MANIFEST": "OpenSpeedTest-Server.json",
```

Comment:
Same remark here, use `openspeedtest-server.json` instead and also rename the corresponding file.